info.json

```diff
@@ -0,0 +1,77 @@
+{
+    "name": "fortiManagerZTPFlow-FeatureExamples",
+    "version": "1.0.9",
+    "fsrMinCompatibility": "7.4.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "FortiManager ZTP Flow - Feature Examples",
+    "dependencies": [
+        {
+            "name": "fortiManagerZTPFlow",
+            "type": "solutionpack",
+            "label": "FortiManager ZTP Flow",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Community",
+    "description": "Feature focused solution examples using the FortiManager ZTP Flow Solution Pack. Create device models in your FMG and apply example solutions to see in action what would happen before you apply solutions to your production gear.",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-feature-examples",
```

Comment:
Please provide a link to README.md on the release/1.0.0 branch. i.e

https://github.com/fortinet-fortisoar/solution-pack-fortimanager-ztp-feature-examples/blob/release/1.0.0/README.md